src/System.Management.Automation/engine/SpecialVariables.cs

```diff
@@ -255,6 +255,11 @@ internal static class SpecialVariables
 
         internal static readonly VariablePath InformationPreferenceVarPath = new VariablePath(InformationPreference);
 
+        internal const string PSNativeCommandUseErrorActionPreference = nameof(PSNativeCommandUseErrorActionPreference);
```

Comment:
This can confuse a bit
```suggestion
        internal const string PSNativeCommandUseErrorActionPreferenceName = nameof(PSNativeCommandUseErrorActionPreference);
```

Reply:
All the other variables are named this way though, so convention has established this